debug_answers.py

```diff
@@ -40,7 +42,8 @@ def divide_by(num, denom):
 # 3. conceptual error and/or typo
 # ---------------------------------
 def bigger_than_ten(x):
-	# return True if x is bigger than 10 else False
+	'''return True if x is bigger than 10 else False'''
+	
```

Comment:
These look good. Could you add the rest of the docstrings below?

Reply:
Yes, absolutely, I missed that.

Reply:
Thanks!